src/ui/components/Timeline/Tooltip.tsx

```diff
@@ -1,37 +1,38 @@
 import React from "react";
-import { connect, ConnectedProps } from "react-redux";
-import { selectors } from "ui/reducers";
-import { UIState } from "ui/state";
+import { useSelector } from "react-redux";
+import { getIsFocusing, getNonLoadingTimeRanges } from "ui/reducers/app";
+import { getHoverTime, getZoomRegion } from "ui/reducers/timeline";
 import { getVisiblePosition } from "ui/utils/timeline";
 
-export function Tooltip({ hoverTime, zoomRegion, timelineWidth, isFocusing }: Props) {
+const getTimestamp = (time?: number) => {
+  const date = new Date(time || 0);
+  const seconds = date.getSeconds().toString().padStart(2, "0");
+  const minutes = date.getMinutes();
+
+  return `${minutes}:${seconds}`;
+};
+
+export default function Tooltip({ timelineWidth }: { timelineWidth: number }) {
+  const hoverTime = useSelector(getHoverTime);
+  const zoomRegion = useSelector(getZoomRegion);
+  const isFocusing = useSelector(getIsFocusing);
+  const nonLoadingRegion = useSelector(getNonLoadingTimeRanges);
   const shouldHideTooltip = !hoverTime || isFocusing;
+
   if (shouldHideTooltip) {
     return null;
   }
 
   let offset = getVisiblePosition({ time: hoverTime, zoom: zoomRegion }) * timelineWidth;
 
-  const time = new Date(hoverTime || 0);
-  const seconds = time.getSeconds().toString().padStart(2, "0");
-  const minutes = time.getMinutes();
+  const isHoveredOnNonLoadingRegion = nonLoadingRegion.some(
+    r => r.start <= hoverTime && r.end >= hoverTime
+  );
+  const msg = isHoveredOnNonLoadingRegion ? "Unloaded" : getTimestamp(hoverTime);
```

Comment:
I'm curious why we just show "unloaded" here vs showing the time _and_ "unloaded" message?

Reply:
I figured @jonbell-lot23 would have input here that we can address as a follow-up. 

For now, I tried to err on making the signal noisier than quieter, and a big `Unloaded` as well as the timestamp missing felt like a nice unmistakable signal to the user that something's up.

On that note, I just realized that if while hovering on the timeline and you saw unloaded, it might scare off the user from clicking it in the first place. Maybe we need to be more explicit with an even louder message, e.g. "Click here to fix it"?